Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,39 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@{{plainid}}\.'
```

Comment:
I'd like it if we could give them this scope, but *overlapping* with the `storage.modifier.annotation` scope, so that the `@` doesn't end up "randomly not colored" in most color schemes.